src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
```

Comment:
The `text-decoration: none;` property is not applicable to a list. It's used for links, which are not present in this block of code.